src/compositor/output/output.rs

```diff
@@ -38,33 +38,37 @@ impl OutputHandler for Output {
 
 /// Render all of the client views.
 fn render_views(renderer: &mut Renderer, layout: &mut OutputLayoutHandle, views: &mut [View]) {
+    with_handles!([(layout: {&mut *layout})] => {
```

Comment:
Don't put that here, that's ugly. `with_handles` takes multiple arguments for this purpose.